modules/programs/borgmatic.nix

```
@@ -27,6 +30,23 @@ let
     description = "Extra settings.";
   };
 
+  repositoryOption = types.submodule {
+    options = {
+      path = mkOption {
+        type = types.str;
+        description = "Path of the repository";
+        example = "ssh://myuser@myrepo.myserver.com/./repo";
+      };
+
+      label = mkOption {
+        type = types.nullOr types.str;
+        default = null;
+        description = "Label of the repository";
```

I would give a little more info here. Maybe something like

```suggestion
        description = "Short text describing the repository. Can be used with the `--repository` flag to select a repository";
```